R/reverse_operons.R

```diff
@@ -23,7 +25,13 @@
 #' straightenOperonSeq(genomic_context)
 #'
 #' # Output: "A->", "B->", "*", "<-C", "<-D", "=", "E->", "F->"
+
 straightenOperonSeq <- function(prot) {
+    # Check if 'prot' is a data frame
+    if (!is.data.frame(prot)) {
+        abort("Error: 'prot' must be a data frame.")
+    }
+    
```

Comment:
Here, the `prot` param should be. a vector. This param gets recycled quite a bit in MolEvolvR, and eventually we should make it consistent across functions or use a new param name.